### PR TITLE
refactor:update-inv-functions

### PR DESCRIPTION
**Pull request by andybeltz**
### Summary

- Enhanced `Export-InvJsonSpec` cmdlet to export additional fields into the json file.

### Type

- [ ] Bugfix
- [X] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [X] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### Test and Documentation

- [X] Tests have been completed.
- [ ] Documentation has been added or updated.

### CODE REVIEW

**Review by GaryJBlake — APPROVED** [review]
LGTM! - Will wait for previous pull to be addressed before merge

**Review by tenthirtyam — APPROVED** [review]

**Pull request merged**
